qoffer-infrastructure/src/main/groovy/life/qbic/Presenter.groovy

```diff
@@ -0,0 +1,21 @@
+package life.qbic
+
+import life.qbic.portal.portlet.customers.create.CreateCustomerOutput
+import life.qbic.portal.portlet.customers.update.UpdateCustomerOutput
+import life.qbic.portal.portlet.offers.create.CreateOfferOutput
+import life.qbic.portal.portlet.offers.search.SearchOffersOutput
+import life.qbic.portal.portlet.offers.update.UpdateOfferOutput
+
+/**
+ *  Handles the presentation of the qoffer-2.0 use cases and its internal data model to the viewModel.
+ *
+ *  This class acts as connector between the use cases of qoffer-2.0 and its viewModel
+ *  and should be used to transfer data between these components.
+ *  It converts the use case output into a presentable data format for the viewModel implementing {@link PortletView}.
```

Comment:
The ViewModel does not implement the Portlet View. It is a stand-alone DTO providing beans to be listened to at most. The `ViewModel` itself is still missing from this PR.

Reply:
Done!

Reply:
True, i kinda got confused sorry:/

Reply:
Sorry but I think it still is not correct.
> It converts the use case output into a presentable data format for the view via implementing {@link PortletView}.
I would suggest something like
`It converts the use case output into a presentable data format for the {@link ViewModel}.`
You also need to create a `ViewModel.groovy` class.

Reply:
You're right again! It's hopefully fixxed now